hack/build/ci/install_go_test_dependencies.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+installCGODependencies() {
```

Comment:
please move each function into a separate script

Reply:
The idea behind this "partial" split into only some scripts was that we have one script for each step in the ci pipeline. But yeah, agree that we should ev. split functions into files if they are called from the outside, and even not only from the ci pipeline.

Reply:
Moved now all the functions into own scripts, besides of some functions in prepareVariablesAndDependencies. But this functions are just there to make the code look better and are not called from outside the script.